Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -71,10 +72,40 @@ std::shared_ptr<grpc::ChannelCredentials> CreateSslCredentials(
   return grpc::SslCredentials(options);
 }
 
-struct HostConfig {
-  util::Path certificate_path;
-  std::string target_name;
-  bool use_insecure_channel = false;
+class HostConfig {
+  using Guard = std::lock_guard<std::mutex>;
+
+ public:
+  void certificate_path(const util::Path& new_value) {
```

Comment:
Nit: there is a using declaration for `Path` already -- no need to specify the namespace.

Reply:
Done